addons/account_edi_proxy_client/data/neutralize.sql

```diff
@@ -3,3 +3,6 @@ INSERT INTO ir_config_parameter (key, value)
 VALUES ('account_edi_proxy_client.demo', true)
     ON CONFLICT (key) DO
        UPDATE SET value = true;
+
+UPDATE account_edi_proxy_client_user
+SET edi_mode = 'demo';
```

Comment:
thank you for fixing this :slightly_smiling_face: 
since now we can have edi users who are not Italian (peppol), where we don't have a demo mode, I think this script should be updated a bit, something like this:
```suggestion
SET edi_mode = CASE
                    WHEN proxy_type='l10n_it_edi' THEN 'demo'
                    ELSE 'test'
               END
```
The lines above with the `'account_edi_proxy_client.demo'` can be deleted I think